SonarQube.TeamBuild.PreProcessor/BuildWrapperInstaller.cs

```diff
@@ -31,7 +33,7 @@ public BuildWrapperInstaller(ILogger logger)
 
         #region IBuildWrapperInstaller methods
 
-        public void InstallBuildWrapper(ISonarQubeServer server, string binDirectory)
+        public AnalysisProperties InstallBuildWrapper(ISonarQubeServer server, string binDirectory, string outputDirectory)
```

Comment:
The _BuildWrapperInstaller_ is now responsible for deciding on any build-wrapper specific settings that should be added to the config file. Currently the only setting is the build wrapper output directory, but if we had to add more in the future then it would be done here.